### ISSUE TITLE
Improve search result presentation

### ISSUE THREAD

**Issue by peer35** [UtrechtUniversity/yoda-ansible#37]
Not really a bug, but could be improved:
- Search for a known file in a research group
- The result shows you the file and a clickable folder name.
- Clicking on the folder takes you to the first page of the file list for that folder, that is not very helpful for folders with large numbers of files.

Expected behavior:      
- The system should send you to the page containing the file you were looking for.

**Comment by lwesterhof** [comment]
Tracked in our internal Issue management system as issue YDA-3917.

**Comment by Danny-dK** [comment]
Is this still an issue when you can increase number of items in the webportal to view up to a 100 (10, 25, 50, 100)?

**Comment by peer35** [comment]
@Danny-dK Yes, the particular project that prompted this has a folder containing over a 1000 photos (of an archaeological excavation). If I search for a particular picture, Yoda shows me that file as a search result, but I can't open or download it from that page. I have to click on the search result link which then drops me on page 1 of the folder. Even with 100 items per page that does not help me if it's number 568.

Come to think of it: it would be even better if the file context menu was available on the search results page, so you could immediately view or download the file.
![image](https://github.com/UtrechtUniversity/yoda/assets/5236397/e5ed4196-1b8d-4223-a3b8-eb7f98ae82d4)
